Library/Homebrew/cmd/install.rb

```diff
@@ -74,7 +74,7 @@ def install_args
           description: "Install testing dependencies required to run `brew test` <formula>.",
         }],
         [:switch, "--HEAD", {
-          description: "If <formula> defines it, install the HEAD version, aka. master, trunk, unstable.",
+          description: "If <formula> defines it, install the HEAD version, aka. main, trunk, unstable.",
```

Comment:
```suggestion
          description: "If <formula> defines it, install the HEAD version, aka. main, trunk, unstable, master.",
```

Reply:
I think it's best to include `master` in here until Git no longer uses it at all.